.spelling

```diff
@@ -380,7 +380,7 @@ L4-L6
 learnings
 LibreSSL
 lifecycle
-LightStep
```

Comment:
I think you'll need to keep both versions in .spelling for now, so that the lint checker won't complain about the translated versions, which are not updated yet.